config/Version.Public.xcconfig

```diff
@@ -1,4 +1,4 @@
 VERSION_SHORT=9.1
 
 // Public long version example: VERSION_LONG=1.2.0.0
-VERSION_LONG=9.1.0.0
+VERSION_LONG=9.1.0.2
```

Comment:
This is `.2` instead of `.1` because https://github.com/woocommerce/woocommerce-ios/pull/6830 didn't get into `trunk` in time. Its changes are included in this PR, of course.

I'm guessing both PRs exist open at the same time because #6830 is based off `release/9.1` while this one off a dedicated `merge/release...` branch. Otherwise, GitHub would have simply updated the existing PR instead of allowing the creation of a new one (_at least, this is what happened to me a few times in the past, if I recall correctly_)